### PR TITLE
Add install.md in test_tipc, and fix some bug

### PR DESCRIPTION
**Pull request by HydrogenSulfate**
1. Add install.md in test_tipc
2. delete the shuffle_channel_detect_pass when construct tensorrt config.
3. delete batchsize=2 case in BMN's tipc

### CODE REVIEW

File: tools/predict.py [hunk 1]
```diff
@@ -99,6 +99,7 @@ def create_paddle_predictor(args, cfg):
     config.enable_memory_optim()
     # use zero copy
     config.switch_use_feed_fetch_ops(False)
+    config.delete_pass("shuffle_channel_detect_pass")
```
Comment (huangjun12) [inline tools/predict.py]: 这个修改的作用是？

Comment (HydrogenSulfate) [reply tools/predict.py]: 去掉会导致STGCN trt模式报错的shuffle_channel_detect_pass

Comment (huangjun12) [reply tools/predict.py]: 这个去掉，会影响其它模型的性能吗

**Comment by huangjun12** [comment]
LGTM

**Pull request merged**